estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Real Estate',
+    'version': '1.0',
+    'author': "Abdelrhman Fawzy",
+    'category': 'Aditi/Real Estate',
+    'summary': "Realistic business advertisements",
+    'application': True,
+    'installable': True,
+    'auto_install': False,
```

Comment:
This is already the default (ref: [doc](https://www.odoo.com/documentation/18.0/developer/reference/backend/module.html?highlight=auto_install))